tests/benchmark/scripts/plotter.py

```diff
@@ -0,0 +1,299 @@
+import os
+import pickle as pkl
+from enum import IntEnum, unique
+
+import matplotlib.pyplot as plt
+import numpy as np
+from json_reader import JSONReader
+from logger import Logger
+
+log = Logger.get()
+
+
+class Plotter:
+    """
+    Plots, saves, displays JSON benchmark results
+    """
+
+    def __init__(self, work_dir, json_reader):
+        self.__json_ids = json_reader.ids()
+        self.__json_num_experiments = json_reader.num_experiments()
+        self.__json_attributes = json_reader.attributes()
+        self.__json_families = json_reader.families()
+        self.__json_measurements = json_reader.measurements()
+        self.__figures = dict()
+        self.__figures_dir = self.__create_directory(work_dir, "figures")
+        self.__pickles_dir = self.__create_directory(work_dir, "bin")
+        self.__figure_id = 0
+
+    @staticmethod
+    def __create_directory(path, dir_name):
+        dir = os.path.join(path, dir_name)
+        if not os.path.isdir(dir):
+            log.info("Creating the work directory: {}".format(dir))
+            os.mkdir(dir)
+        return dir
+
+    def __exists(self, figure_id):
+        """
+        Checks if fig exists in dict
+        """
+        return True if figure_id in self.__figures.keys() else False
```

Comment:
you do not need keys() here

Reply:
replaced by `return figure_id in self.__figures`